client/settings/survey-modal/style.scss

```diff
@@ -0,0 +1,35 @@
+.survey-section {
```

Comment:
(minor) Seeing some inconsistent spacing between the elements - what do you think about giving them some breathing room?

```
.survey-section {
	.comments-text-field {
		margin-top: 2em;

		label {
			font-weight: 600;
		}

		textarea {
			height: 4rem;
		}
	}

	.disable-success-notice {
		background: #e6f2e8;
		padding: 16px;
		border-radius: 4px;

		p {
			display: inline;
			margin-left: 20px;
		}

		.disable-success-icon {
			display: inline;
			color: #4ab866;
		}
	}

	.survey-bottom-disclaimer {
		font-style: italic;
	}
}

```